config/forms/page_block_settings.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" ?>
+<form xmlns="http://schemas.sulu.io/template/template"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/form-1.0.xsd"
+>
+    <key>page_block_settings</key>
+
+    <properties>
+        <section name="custom">
+            <properties>
+                <property name="theme" type="single_select">
+                    <meta>
+                        <title lang="en">Block Theme</title>
+                        <title lang="de">Block Theme</title>
+                    </meta>
+
+                    <params>
+                        <param name="default_value" value="default"/>
+
+                        <param name="values" type="collection">
+                            <param name="default">
```

Comment:
When https://github.com/sulu/sulu/pull/ is merged, the following works too. I think this would be better because we dont add a `blocks__item--default` class (which is only added after the block settings were opened). 

```suggestion
                        <param name="default_value" value=""/>

                        <param name="values" type="collection">
                            <param name="">
```